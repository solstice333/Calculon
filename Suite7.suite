P a.out compile.c
T test11.in test11.out 1000
P b.out compile.c
T test11.in test11.out 1000
