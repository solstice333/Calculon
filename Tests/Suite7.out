make: *** No rule to make target `doesntexist.c', needed by `a.out'.  Stop.
Failed: make a.out
compile.c:1: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘is’
compile.c:2: error: expected identifier or ‘(’ before ‘<’ token
compile.c:2: error: stray ‘#’ in program
compile.c:3: error: expected identifier or ‘(’ before ‘++’ token
compile.c:3:3: error: empty character constant
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
make: *** [b.out] Error 1
Failed: make b.out
